packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/method_channel_document_reference.dart

```diff
@@ -65,20 +62,20 @@ class MethodChannelDocumentReference extends DocumentReferencePlatform {
 
   @override
   Future<DocumentSnapshotPlatform> get(
-      [GetOptions options = const GetOptions()]) async {
+      [GetOptions? options = const GetOptions()]) async {
```

Comment:
why is `options` nullable if it has a default value?

Reply:
I get an error because it ceases to be a valid override of its superclass. I'll also change the superclass [here](https://github.com/FirebaseExtended/flutterfire/blob/master/packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/platform_interface/platform_interface_document_reference.dart#L69) so that it works.